### ISSUE TITLE
Scrum update 3

### ISSUE THREAD

**Issue by DSep** [CourseFlo/Flo#17]
- I primarily worked on the Browse page. I broke the page into a page and subcomponents, while also porting the redux to the necessary child subcomponents. I also designed and created the Course type and associated data types stored within, to be used through out the app, including most pages as well as in data retrieval and comms with the backed. Secondarily, I supported theming on the other pages, while other team members made changes or refactored existing code, including supporting addition of redux to navbar and visualizer page.
- It was definitely a challenge to produce our vision into reality. To be honest, we still have a ways to go till the styling, and I think we have all struggled to try to make what we envision encoded into a clean styling. I definitely struggled to customize the material UI theme, without forcing ultra specific styling too much, and also struggled to figure out how a centralized styling Theme would work in Material UI. 
- Most of our tasks are to be determined at our upcoming sprint planning session. For now, I can guess that sprint I will be primarily working on the backend creation and some of the logic that will get courses data from the UBC Courses API, endpoints for the front end to provide it with services like search and get exact course and user data, and universalize some Course types across the frontend and backend that will be used in this communication. (The sending and getting from the database is to be assigned, I might help with this.) At the same time, I will be producing a better course Results component and refactoring the Search redux action, to incorporate the Course data type and the API call + response involved in performing a _Search_
CSID: g3u1b

**Comment by yqj2k** [comment]
- I worked on the ProfilePage of the application which includes the user's personal information, the courses they're taking, and implemented state persisting of the user's information.
- I ended up trying to centralize our styling for the application for a few days and did not go anywhere for a few days. I ended up being unable to centralize the styling for our whole application which put the work back I could do. Additionally, I am finishing up a course for my first term which adds to my workload (on-top of working full-time).
- I will work on connecting the front-end to backend for the user information ( creating user endpoints, schema's/tables for mongoose, and async calls in the frontend)
csid: a4g2b

**Comment by baconandchips** [comment]
- I worked on the VisualizeCourse page of the application, where I tried to implement Redux into our pages. After being unable to do so, I implemented Material-UI into the page here. 
- My progress is not up to my own standards; I feel that a huge part is that I overestimated the task in my head. Wasn't a good idea. The team gave suggestions on doing work in incremental progress, and I will work hard to rectify my mistake in this sprint onwards. 
- In the next two/three weeks, I will be tackling Material-UI central theme first. This is something that I'll do with a blend of tutorial-following and project-building at the same time, and I will be demoing what I have with my team on Friday, so at the very least I'll be seeking early intervention if I still encounter issues by that time, and in a better case scenario I'll complete the task successfully and be able to move onto the next steps for improving the codebase, presumably making sure that Redux works smoothly in my pages. 
csid: n0k0b

**Comment by beetai** [comment]
- I worked on reworking the login page such that it would use material UI. A signin page has also been created alongside it with very basic styling. Additionally, I created the Navbar and attached a login action to it with Sepand’s help. I have also done some research for MongoDB to understand all its basics and prepare myself to add it to the project.
- Something I always find myself struggling with is continuing work once I find myself stuck, though with the help of my teammates, I have learned that it really helps to ask for help from them at the right times. Pair programming has really helped a lot of is pull through at times. Another struggle for me is keeping up with understanding how to approach Material UI theming, since it seems important to have this working from a UI/UX point of view. 
- I plan to work on parsing UBC course information to add it to our database. Additionally, I want to contribute in linking up the backend to our frontend using Express, with packages such as Mongoose and Axios to help us out.
csid: c8m1b
